src/data-dictionary/events/Deployment/entity.guid.md

```diff
@@ -0,0 +1,8 @@
+---
+name: entity.guid
+type: attribute
```

Comment:
```suggestion
type: attribute
units: ID
```

Reply:
@zuluecho9 the entity.guid is actually not an ID field. It is a free-form field (according to our SME).